pixel.js

```diff
@@ -135,6 +137,7 @@ function setupCli() {
 		.description( 'Create test screenshots and compare them against the reference screenshots.' )
 		.requiredOption( ...branchOpt )
 		.option( ...changeIdOpt )
+		.option( '--mobile' )
```

Comment:
Can a description of the option be added (second param)?

Reply:
Done.